CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Release process
+
+At the time of writing we have a pretty informal release process for the packages in this repo. There are obvious improvements that we can and should make, but for now we're starting by documenting the current process. By way of example, here are the steps taken to publish updated packages based on [ff680bb9ebbee](https://github.com/liferay/liferay-npm-tools/commit/ff680bb9ebbee43711bb7bf03d3e852716c54616):
+
+
+```sh
+# Make sure the local "develop" branch is up-to-date:
+git checkout develop
+git pull --ff-only upstream develop
+
+# Make sure formatting is up-to-date:
+yarn format:check
+
+# Run tests (currently, we don't have many of them):
+(cd packages/liferay-jest-junit-reporter && yarn test)
+
+# Update individual package versions:
+# - Note that this time we updated all the packages,
+#   but on many occasions we'll update only one, or two.
+cd packages/liferay-jest-junit-reporter
+yarn version --no-git-tag-version --new-version 1.0.1
```

Comment:
Didn't know about this, I always use `npm version [patch|minor|major]`

Reply:
Yeah this is just me being excessively explicit out of paranoia, because `yarn version` has `--minor`, `--patch` etc flags too, along with [a bunch of other features](https://yarnpkg.com/lang/en/docs/cli/version/).